packages/agoric-cli/package.json

```diff
@@ -35,10 +35,10 @@
     "@agoric/cosmic-proto": "^0.2.1",
     "@agoric/ertp": "^0.15.3",
     "@agoric/internal": "^0.2.1",
-    "@endo/nat": "^4.1.0",
     "@agoric/smart-wallet": "^0.4.2",
     "@agoric/swingset-vat": "^0.30.2",
     "@agoric/zoe": "^0.25.3",
+    "@confio/relayer": "^0.7.0",
```

Comment:
We're going to add a whole IBC relayer to our dependencies?

Should we talk this over a bit? Should we have an issue?

Reply:
This relayer is a thin veneer on top of cosmjs.  If it's dragging in too much, we can probably dedupe some cosmjs versions.